pkgs/by-name/qu/qubes-vmm-xen/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchpatch,
+  buildXenPackage,
+  qubes-seabios,
+  xen,
+}:
+
+let
+  version = "4.19.0";
+  revision = "3";
+
+  qubesPatches = import ./patches.nix {
+    inherit fetchpatch version revision;
+  };
+  qubesPatchList = lib.lists.flatten (
+    with qubesPatches;
+    [
+      EFI_WORKAROUNDS
+      BACKPORTS
+      UPSTREAMABLE_PATCHES
+      QUBES_SPECIFIC_PATCHES
+      OTHERS
+    ]
+  );
+in
+
+buildXenPackage.override
```

Comment:
For Qubes, I also add overrideAttrs with installPhase to copy qubes-vmm-stubdom binaries here, any idea on how to perform this operation in a cleaner maner?

Reply:
qubes-vmm-xen also needs a corresponding python module

Reply:
Feel free to pick https://github.com/CertainLach/nixpkgs/commit/9f6e6ce1e8cf82396dd9d7f56e7b751267841a09 for python module.